src/components/duelling-picklist/picklist-item/picklist-item.style.ts

```diff
@@ -35,6 +36,18 @@ const StyledPicklistItem = styled.li<{ locked?: boolean }>`
 `;
 
 const StyledButton = styled(Button)`
+  border-bottom-right-radius: var(--borderRadius100);
```

Comment:
As Button has some styling targeting the focus state you'll need to override the left top and bottom radiuses here as well as it's currently not correct

```
 :focus {
    border-bottom-left-radius: var(--borderRadius000);
    border-top-left-radius: var(--borderRadius000);
  }
```
![image](https://user-images.githubusercontent.com/44157880/228313855-3724d491-39e8-4b34-9858-ac8e70d447c1.png)